packages/terra-action-header/src/ActionHeader.jsx

```diff
@@ -88,10 +106,25 @@ const ActionHeader = ({
   onPrevious,
   onNext,
   children,
+  backButtonA11yLabel,
+  nextButtonA11yLabel,
+  prevButtonA11yLabel,
   ...customProps
 }) => {
   const theme = React.useContext(ThemeContext);
 
+  const closeButtonId = `terra-action-header-close-button-${uniqueid()}`;
+  const maximizeButtonId = `terra-action-header-maximize-button-${uniqueid()}`;
+  const minimizeButtonId = `terra-action-header-minimize-button-${uniqueid()}`;
+
+  const visuallyHiddenComponent = (
+    <>
+      <VisuallyHiddenText aria-hidden id={closeButtonId} text={intl.formatMessage({ id: 'Terra.actionHeader.close.description' }, { text })} />
+      <VisuallyHiddenText aria-hidden id={maximizeButtonId} text={intl.formatMessage({ id: 'Terra.actionHeader.maximize.description' }, { text })} />
+      <VisuallyHiddenText aria-hidden id={minimizeButtonId} text={intl.formatMessage({ id: 'Terra.actionHeader.minimize.description' }, { text })} />
```

Comment:
don't we need similar descriptive texts for Back button, Next button and Previous button.

Reply:
The descriptive texts will be passed as props for these Back, Next and Prev buttons to icon Button's text prop. Since we are not sure of the context in which these buttons will be used.